pkg/agent/proxy/proxier_test.go

```diff
@@ -17,27 +17,58 @@ package proxy
 import (
 	"fmt"
 	"net"
+	"runtime"
 	"testing"
 	"time"
 
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
+	apiruntime "k8s.io/apimachinery/pkg/runtime"
 	apimachinerytypes "k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/component-base/metrics/testutil"
 
+	"antrea.io/antrea/pkg/agent/config"
 	"antrea.io/antrea/pkg/agent/openflow"
 	ofmock "antrea.io/antrea/pkg/agent/openflow/testing"
 	"antrea.io/antrea/pkg/agent/proxy/metrics"
 	"antrea.io/antrea/pkg/agent/proxy/types"
+	"antrea.io/antrea/pkg/agent/route"
+	routemock "antrea.io/antrea/pkg/agent/route/testing"
 	binding "antrea.io/antrea/pkg/ovs/openflow"
 	k8sproxy "antrea.io/antrea/third_party/proxy"
 )
 
+var gwMAC, _ = net.ParseMAC("bb:bb:bb:bb:bb:bb")
+var gwIPv4, gwIPv6 = net.ParseIP("192.168.250.1"), net.ParseIP("aa:aa:aa:aa:aa:aa:aa:aa")
+var gwConfigV4 = config.GatewayConfig{
+	Name:      "antrea-gw0",
+	IPv4:      gwIPv4,
+	MAC:       gwMAC,
+	LinkIndex: 50,
+}
+var gwConfigV6 = config.GatewayConfig{
+	Name:      "antrea-gw0",
+	IPv6:      gwIPv6,
+	MAC:       gwMAC,
+	LinkIndex: 50,
+}
+var gwConfigMap = map[bool]*config.GatewayConfig{
+	false: &gwConfigV4,
+	true:  &gwConfigV6,
+}
+var gwIPMap = map[bool]net.IP{
+	false: gwIPv4,
+	true:  gwIPv6,
+}
+
+func isWindows() bool {
+	return runtime.GOOS == "windows"
+}
```

Comment:
maybe you can use the existing `IsWindowsPlatform` function (pkg/util/runtime)

Reply:
Updated.